packages/gatsby-theme-newrelic/gatsby/on-pre-render-html.js

```diff
@@ -1,11 +1,42 @@
 import React from 'react';
 import path from 'path';
+import { getGtmConfig } from '../src/utils/config';
 import { TESSEN_PATH } from './constants';
 
 const onPreRenderHTML = (
   { getHeadComponents, replaceHeadComponents },
   themeOptions
 ) => {
+  const googleTagManager = getGtmConfig(themeOptions);
+  let gtagScript;
+  let googleTrackScript;
+
+  if (themeOptions.googleTagManager) {
+    gtagScript = (
```

Comment:
[nit] It doesn't look like these really need to be in a conditional since you're checking them anyways [on line 61 and 62](https://github.com/newrelic/gatsby-theme-newrelic/pull/343/files#diff-2ce4e67e6fc3cf9f2b0afa489dce365fa84ce3014b74049c9f9e5eda2556f4daR61). I'd just pull these out to a `const` and remove this conditonal.